pom.xml

```diff
@@ -1441,8 +1441,8 @@
     <powermockVersion>2.0.9</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>0.5.1</opennmsApiVersion>
-    <opennmsApiVersionOsgi>0.5.1</opennmsApiVersionOsgi>
+    <opennmsApiVersion>0.6.0-SNAPSHOT</opennmsApiVersion>
```

Comment:
TODO: use fixed version once its published

Reply:
I published a release version of 0.6.0 - it should be available and indexed within the next hour or so.

Reply:
I updated it.